src/figuresClasses.ts

```diff
@@ -1,11 +1,106 @@
-export interface Figure {}
+type Colors = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+enum Shapes {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
```

Comment:
[CODE STYLE] - Enum constant names usually start with a capital letter. In the `Shapes` enum, the values should start with a capital letter to follow TypeScript conventions.

GOOD EXAMPLE:
enum Shapes {
  Triangle = 'TRIANGLE',
  Circle = 'CIRCLE',
  Rectangle = 'RECTANGLE',
}